api/api.yml

```diff
@@ -401,6 +472,8 @@ paths:
       - machine manager
       summary: get all requests
       operationId: getRequests
+      security:
+      - OAuth2: [administrator]
```

Comment:
Myślałem, że weryfikacja jest automatyczna...

Reply:
A taka powinna być? Jeśli automatyczna to nie powinienem wystawiać endpoibntów żadnych

Reply:
Owszem